### PR TITLE
added function localize_url() to web/util/helpers

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: src/clld/web/util/helpers.py [hunk 1]
```diff
@@ -711,3 +711,26 @@ def get_resource_type(obj):
...
+
+    :param req: current request
+    :param locale: language code (ISO 639-1:2002)
+    :param default_locale: default language code (ISO 639-1:2002)
+    :return: URL as string
+    """
+    assert locale, 'Missing locale'
+    k_str = '__locale__'
+    p_url = urlsplit(req.url)
+    params = req.query_params
```
[the remark this answers is not in the document]
Comment (Bibiko) [reply src/clld/web/util/helpers.py:728]: ah - good point

[the remark this answers is not in the document]
Comment (Bibiko) [reply src/clld/web/util/helpers.py:728]: but then, one has to give up the default_locale approach

[the remark this answers is not in the document]
Comment (xrotwang) [reply src/clld/web/util/helpers.py:728]: Why? In the line I've given above, the `__locale__` key isn't copied if the value is the default encoding.

[the remark this answers is not in the document]
Comment (Bibiko) [reply src/clld/web/util/helpers.py:728]: it must be `params = {k: v for k, v in req.query_params.items() if not (k == k_str and locale.lower() == default_locale)}` then it works

**Review by xrotwang — APPROVED** [review]

**Pull request merged**